project-koku/koku
# Add IBM Cloud Provider

This is essential for starting with the IBM Cloud Downloader.

# How to test it
What I personally did to make sure it works is adding IBM source(s) to `./scripts/test_customer.yaml`.
This file contains all the sources that are created using `make create-test-customer`.

Add under `sources`:
```
---
customer:
  account_id: 10001
  email: test@example.com
  user: test_customer
  sources:
    - ibm_source:
      source_name: Test IBM Source
      source_type: 'IBM'
      authentication:
        credentials:
          iam_token: 'yadayadayoow'
      billing_source:
        data_source:
          enterprise_id: 'blablabla'
    ......
```

You should see in `koku_server` that this is failing due to bad API token.
Ping me so I can give you a valid API Token and Enterprise ID to verify that it works when the data is right.
